Header header
cob_perception_msgs/DetectionArray detections
string detector
